pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -4,13 +4,13 @@ with python3Packages;
 
 buildPythonApplication rec {
   pname = "linkchecker";
-  version = "unstable-2020-08-15";
+  version = "10.0.0";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
-    rev = "0086c28b3a419faa60562f2713346996062c03c2";
-    sha256 = "0am5id8vqlqn1gb9jri0vjgiq5ffgrjq8yzdk1zc98gn2n0397wl";
+    rev = "v" + version;
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
I get the latter is more commonly seen in nixpkgs, but otherwise why?

Reply:
If we ever do treewide changes on this it would make it a lot easier to change when we don't need to think about this variant.

Reply:
I'm not really convinced. The revision is inherently specific to *this* derivation.